docs/index.md

```diff
@@ -28,6 +28,9 @@
   </br></br>
 </span>
 
+## ✨ [Try it in your browser](https://scipp.github.io/toflite/lab/index.html?path=app.ipynb) ✨
```

Comment:
Ok, but maybe people will be confused by the icon which is nowadays probably associated with AI?

Reply:
Hah, then maybe keep it!